pkg/cassandra/config/config.go

```diff
@@ -134,8 +173,31 @@ type SessionBuilder interface {
 	NewSession() (cassandra.Session, error)
 }
 
+func (c *Configuration) newSessionPrerequisites() error {
+	if !c.Schema.CreateSchema {
+		return nil
+	}
+	cluster, err := c.NewCluster()
```

Comment:
check error

Reply:
actually, all this code effectively repeats `NewSession`, the only difference is the keyspace. Can you move it into a helper `createSession(keyspace string)`?

Reply:
You can make the new function to depend on the config argument, and pass whichever config is needed in each instance, including cloning it to override keyspace